src/components/PostsContainer/PostsPage.js

```diff
@@ -1,18 +1,23 @@
 //Complete the necessary code in this file
 // import useState
-import React from "react";
+import React, { useState } from "react";
 import Post from "./Post";
 import "./Posts.css";
-// import data 
+// import data
+import dummyData from "../../dummy-data";
 
 const PostsPage = () => {
   // set up state for your data
+  const [post, setPost] = useState(dummyData);
   return (
     <div className="posts-container-wrapper">
-      {/* map through data here to return a Post and pass data as props to Post */}
+      {dummyData.map(item => {
```

Comment:
Remember to use your state rather then your import when rendering so `post.map()` rather then `dummyData.map()` /